usbl_seatrac.orogen

```diff
@@ -25,6 +25,7 @@ task_context "Task", subclasses: "iodrivers_base::Task" do
 
     output_port("pose", "base/samples/RigidBodyState")
     output_port("ping_status", "usbl_seatrac/PingStatus")
+    output_port("local_pressure", "uint32_t")
```

Comment:
Never *ever* output a reading from a sensor without timestamping.

There is a `base/samples/Pressure` type